client/performance-profiler/components/charts/history-chart.jsx

```diff
@@ -236,13 +226,7 @@ const HistoryChart = ( { data, range, height } ) => {
 				<svg ref={ svgRef }></svg>
 				{ ! dataAvailable && (
 					<div className="info">
-						History unavailable
-						<Icon
-							onMouseOver={ handleInfoToolTip }
-							onMouseOut={ handleHideToolTip }
-							icon={ info }
-							className="icon"
-						/>
+						{ translate( 'Historical data will appear after more test runs' ) }
```

Comment:
I just remembered that this is not true. It won't appear automatically unless we make changes on the backend. I believe we haven't planned that yet.

Reply:
So just to clarify, we will only be able show the history when the site has CrUX data, right?

Reply:
Exactly. It will only be available when CruX data is available.

Reply:
Okay, in that case, until we iterate on this, I think we should update the copy to suggest that there's no historical data available for the specific site. @matt-west wdyt would be the most appropriate copy here?

Reply:
Sorry, I just saw your message here: p1726048080075789/1725969661.316119-slack-C07AV9N4Y1H